parser/parser.go

```diff
@@ -89,3 +90,8 @@ func (p *Parser) Parse(fname string, isDir bool) error {
 	}
 	return nil
 }
+
+func getDefaultGoPath() (string, error) {
+	output, err := exec.Command("go", "env", "GOPATH").Output()
```

Comment:
I sorry to commenting after the PR was merged, but I'm not sure this change should be made this way. What would happen if go is not in the user's `$PATH`?

It seems that `go/build.Default.GOPATH` is the most appropriate way of getting `GOPATH` since go1.8.